presto-docs/src/main/sphinx/release/release-300.rst

```diff
@@ -0,0 +1,112 @@
+===========
+Release 300
+===========
+
+General Changes
+---------------
+
+* Fix correctness issue for :func:`array_intersect` and :func:`array_distinct`
+  when input contains both zeros and nulls.
+* Fix ``count(*)`` aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
+* Improve the table scan performance for structured types.
+* Improve performance for :func:`array_intersect`.
+* Improve performance of window functions by filtering partitions early.
+* Add :func:`reduce_agg` aggregate function.
+* Add :func:`millisecond` function.
+* Remove ``ON`` keyword from :doc:`/sql/show-stats` (use ``FOR`` instead).
+* Restrict ``WHERE`` clause in :doc:`/sql/show-stats` to filters that can be pushed down to connectors.
+* Return final results to clients immediately for failed queries.
+
+JMX MBean Naming Changes
+------------------------
+
+* The base domain name for server MBeans is now ``presto``. The old names can be
+  used by setting the configuration property ``jmx.base-name`` to ``com.facebook.presto``.
+* The base domain name for the Hive, Raptor, and Thrift connectors is ``presto.plugin``.
+  The old names can be used by setting the catalog configuration property
+  ``jmx.base-name`` to ``com.facebook.presto.hive``, ``com.facebook.presto.raptor``,
+  or ``com.facebook.presto.thrift``, respectively.
+
+Web UI
+------
+
+* Fix rendering of live plan view for queries involving index joins.
+
+JDBC Driver Changes
+-------------------
+
+* Change driver class name to ``io.prestosql.jdbc.PrestoDriver``.
+
+System Connector Changes
+------------------------
+
+* Remove ``node_id`` column from ``system.runtime.queries`` table.
+
+Hive Connector Changes
+----------------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix corner case where the ORC writer fails with integer overflow when writing
+  highly compressible data using dictionary encoding (:issue:`x11930`).
+* Fail queries reading Parquet files if statistics in those files are corrupt
+  (e.g., min > max). To disable this behavior, set the configuration
+  property ``hive.parquet.fail-on-corrupted-statistics``
+  or session property ``parquet_fail_with_corrupted_statistics`` to false.
+* Add support for :ref:`s3selectpushdown`, which enables pushing down
+  projections and predicates into S3 for text files.
```

Comment:
Instead of "projections and predicates", maybe say "column selection and range filters" since it's a little more precise.